lux-lib/src/path.rs

```diff
@@ -2,23 +2,39 @@ use itertools::Itertools;
 use serde::Serialize;
 use std::{env, fmt::Display, io, path::PathBuf, str::FromStr};
 
-use crate::{build::utils::lua_lib_extension, tree::Tree};
+use crate::{
+    build::utils::lua_lib_extension,
+    config::{Config, LuaVersion},
+    tree::Tree,
+};
 
 const LUA_PATH_SEPARATOR: &str = ";";
+const LUA_INIT: &str = "require('lux').loader()";
```

Comment:
question: :thinking: [should we name this module something else](https://github.com/IgorMael/Lux/blob/master/src/lux.lua)?
`lx` isn't taken.

Reply:
I think lux is fine. I'm all for preventing conflicts, but none of the existing rocks are large enough for us to realistically have to worry imo